Kickstarter-iOS/Views/Controllers/MessageThreadsViewController.swift

```diff
@@ -11,9 +11,16 @@ internal final class MessageThreadsViewController: UITableViewController {
   @IBOutlet fileprivate weak var footerView: UIView!
   @IBOutlet fileprivate weak var mailboxLabel: UILabel!
 
-  internal static func configuredWith(project: Project?) -> MessageThreadsViewController {
+//  internal static func configuredWith(project: Project?) -> MessageThreadsViewController {
```

Comment:
Any reason to keep the commented out code around?

Reply:
oh forgot to delete! thanks